roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -153,10 +153,11 @@
 - name: rac-db-install | opatch output
   debug:
     msg:
-      - "{{ apply_oneoff.cmd }}"
-      - "{{ apply_oneoff.stdout_lines }}"
+      - "{{ item.cmd }}"
+      - "{{ item.stdout_lines }}"
   with_items: "{{ apply_oneoff.results }}"
   when: item.changed
+  ignore_errors: yes
```

Comment:
Now that items have been fixed, I don't think we need to ignore errors anymore;  we can expect every line to have a cmd and stdout lines.

Reply:
The o/p for the same Ansible block differed in 12cR2 install vs a 19c install.
We don't want output display to stop the entire install, if for any reason, the differing outputs  (now or in a future version/patch) triggers an error.